payments-ui-core/src/main/java/com/stripe/android/ui/core/elements/H4Text.kt

```diff
@@ -12,10 +13,12 @@ fun H4Text(
     text: String,
     modifier: Modifier = Modifier,
 ) {
-    Text(
-        text = text,
-        color = PaymentsTheme.colors.material.onSurface,
-        style = PaymentsTheme.typography.h4,
-        modifier = modifier
-    )
+    PaymentsTheme {
```

Comment:
Why does this one call PaymentsTheme but H6 below doesn't? Should it be calling?